woodwork/statistics_utils.py

```diff
@@ -300,6 +300,30 @@ def _get_mutual_information_dict(dataframe, num_bins=10, nrows=None, include_ind
     return mutual_info
 
 
+def _get_valid_mi_columns(dataframe, include_index=False):
+    """Returns a list of columns from the DataFrame with valid
+        Logical Types that support mutual information.
+
+    Args:
+        dataframe (pd.DataFrame): Data containing Woodwork typing information
+            from which to calculate mutual information.
+        include_index (bool): If True, the column specified as the index will be
+            included as long as its LogicalType is valid for mutual information calculations.
+            If False, the index column will not have mutual information calculated for it.
+            Defaults to False.
+
+    Returns:
+        list: A list of column names that have valid Logical Types that support
+        mutual information.
+    """
+    valid_types = tuple(get_valid_mi_types())
+    valid_columns = [col_name for col_name, col in dataframe.ww.columns.items() if type(col.logical_type) in valid_types]
```

Comment:
Thanks for making the `valid_types = tuple(get_valid_mi_types())` change.  Were you also going to make the `isinstance(col.logical_type, valid_types)` change?